webapp/package.json

```diff
@@ -51,6 +51,51 @@
     "webpack-cli": "5.0.2",
     "webpack-dev-server": "4.13.3"
   },
+  "overrides": {
+    "@floating-ui/react": {
```

Comment:
how did we got this list of fixed dependency ?

Reply:
I got it through trial and error 😛

If you delete the `package-lock.json` and do an `npm install`, it'll error out on these dependencies because it can't find a version of React or Styled Components which don't satisfy the `peerDependencies` of everything. I just did that repeatedly while slowly adding things in until it worked